_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -512,12 +553,40 @@ The ResponseWrapper is a wrapper around the Response that is sent by the recipie
 
+## Notification
+
+Notifications provide you with the possibility to notify a peer about something, e.g. the succession of one of your Attributes.
+However, unlike [Requests](#request) they don't offer the option to make a decision whether or not they want to accept this change.
+In the example of Attribute succession, the peer can not decide to accept or reject the updated value, but is simply informed about it.
+
+| Name  | Type                                       | Description                                                                                                        |
+| ----- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------ |
+| @type | `"Notification"`                           |                                                                                                                    |
+| id    | `string`                                   | {% include descr_id class="Notification" prefix="NOT" %}                                                           |
+| items | [`NotificationItem[]`](#notificationitems) | An array of NotificationItems that are part of the Notification. There must be at least one Item per Notification. |
+
+### NotificationItems
+
+NotificationItems are sent inside a Notification and specify which processes should be triggered when receiving the Notification.
+
+#### PeerSharedAttributeSucceededNotificationItem
+
+A PeerSharedAttributeSucceededNotificationItem will be sent, if an Attribute, a peer shared with you, was succeeded by them and they choose to notify you about it.
+Internally, the succeeded version will then be created at your side as successor for your previously received Attribute.
+
+| Name             | Type                                                                                           | Description                                        |
+| ---------------- | ---------------------------------------------------------------------------------------------- | -------------------------------------------------- |
+| @type            | `"PeerSharedAttributeSucceededNotificationItem"`                                               |                                                    |
+| predecessorId    | `string`                                                                                       | The `id` of the LocalAttribute that was succeeded  |
+| successorId      | `string`                                                                                       | The `id` of the LocalAttribute it was succeeded by |
+| successorContent | [`IdentityAttribute`](#identityattribute) \| [`RelationshipAttribute`](#relationshipattribute) | The updated content of the LocalAttribute          |
```

Comment:
I think we should use the formatting `...` when refering to properties of a data object. For example, I'd use "`content` of the LocalAttribute" here.